docs/config.rst

```diff
@@ -136,7 +136,8 @@ Optional settings
     .. code-block:: ruby
 
         Raven.configure do |config|
-          config.sanitize_data_for_request_methods -= ["POST", "PUT", "PATCH"] # Do this to send POST, PUT PATCH data
+          config.sanitize_data_for_request_methods = %w(PATCH) # Only send data for PATCH request method
+          # config.sanitize_data_for_request_methods = [] # Do this to always send request data
```

Comment:
I think if someone wants to always send request data, they should just remove the processor altogether: `config.processors -= [Raven::Processors::RequestMethodData]`.